config/routes.rb

```diff
@@ -75,7 +75,9 @@
         resources :audio_files, except: [:new, :edit]
 
         resources :accounts, only: [:index, :show], module: :auth do
-          resources :stories, only: [:index, :create, :update]
+          resources :stories, only: [:index, :create, :update] do
+            get 'search', on: :collection
```

Comment:
This got me thinking about pros/cons of having separate search endpoints vs query params.  But I think we already settled that in slack.  Plus we're not really stuck with anything until we publicize the `/search` endpoints in the links.